src/lib/hooks.js

```diff
@@ -137,7 +137,9 @@ const api = {
 			api.setDepartment(data.department);
 		}
 
-		createOrUpdateGuest(data);
+		createOrUpdateGuest(data).catch((e) => {
```

Comment:
When this fails, are we good just loggin the message to the console? 👀 or should we put an indicator on the ui too? (not sure if this happens already)